devicetypes/shinasys/sihas-usm-300-z.src/sihas-usm-300-z.groovy

```diff
@@ -0,0 +1,278 @@
+/**
+ *  SiHAS USM-300-Z
+ *
+ *  Copyright 2020 Shina System Co., Ltd
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ */
+
+
+metadata {
+	definition (name: "SiHAS USM-300-Z", namespace: "shinasys", author: "Shina System Co., Ltd", cstHandler: true, vid: "generic-motion-6") {
+		capability "Battery"
+		capability "Motion Sensor"
+		capability "Temperature Measurement"
+		capability "Illuminance Measurement"
+		capability "Relative Humidity Measurement"
+        capability "Health Check"
+        capability "Sensor"
+        capability "Refresh"
+        
+        fingerprint profileId: "0104", deviceId: "0001", inClusters: "0000,0001,0003,0004,0020,0400,0402,0406,0500,0B05", outClusters: "0003,0004,0019", manufacturer: "ShinaSystem", model: "USM-300Z", deviceJoinName: "SiHAS USM-300Z"
```

Comment:
Change `deviceJoinName` to `SiHAS  Multipurpose Sensor`

Reply:
In the method you requested, I will do it again later.